List special characters allowed in admin password (#5488)
centreon/centreon-documentation: versioned_docs/version-24.10/installation/web-and-post-installation.md

### Step 5: Admin information

> Define the information needed to create the default admin account. This is the account you will use to log in to Centreon for the first time. The password must comply with the default password security policy: at least 12 characters, lower and upper case letters, numbers and special characters (i.e., @$!%*?&). You can change this policy later.

Author: ```suggestion
Define the information needed to create the default admin account. This is the account you will use to log in to Centreon for the first time. The password must comply with the default password security policy: at least 12 characters, lower and upper case letters, numbers and special characters (at least one of these: @$!%*?&). You can change this policy later.
```
